README.md

```diff
@@ -154,12 +154,12 @@ in the Scorecard GitHub Action setting.
 
```

Comment:
While we're editing the readme, can you also point our badge to the new visualizer?

Reply:
Sure can!